docker-compose.yml

```diff
@@ -239,28 +261,21 @@ services:
         - IDAM_BASE_URL=http://idam-api:8080
         - IDAM_USER_URL=http://idam-api:8080
         - IDAM_LOGOUT_URL=https://localhost:8000/login/logout
+        - IDAM_OAUTH2_CLIENT_ID=ccd_gateway
+        - IDAM_OAUTH2_CLIENT_SECRET=123456
         - IDAM_OAUTH2_TOKEN_ENDPOINT=http://idam-api:8080/oauth2/token
         - IDAM_S2S_URL=http://service-auth-provider-api:8080
         - IDAM_SERVICE_KEY=AAAAAAAAAAAAAAAA
         - IDAM_SERVICE_NAME=ccd_gw
```

Comment:
this may be not required anymore ?

Reply:
still [here](https://github.com/hmcts/ccd-api-gateway/blob/b11caa87a0d6bbd6484ba5c0b5c9868dad6a9918/config/custom-environment-variables.yaml#L12)